src/components/Emphasis.js

```diff
@@ -1,8 +1,12 @@
 import React from 'react';
-import { getNestedValue } from '../utils/get-nested-value';
+import ComponentFactory from './ComponentFactory';
 
 const Emphasis = ({ nodeData }) => (
-    <em>{getNestedValue(['children', 0, 'value'], nodeData)}</em>
+    <em>
+        {nodeData.children.map((child, index) => (
+            <ComponentFactory nodeData={child} key={index} />
```

Comment:
`ComponentFactory` gives the same behavior if the child is of type `text`. This continues to keep the existing functionality for Snooty and Strapi.